oblib/taxonomy_loader.py

```diff
@@ -416,7 +506,10 @@ def _load_elements_file(self, pathname):
         parser.parse(open(pathname))
         return eh.elements()
 
-    def _load_elements(self):
+    def _load_entrypoints_concept_details(self):
```

Comment:
Rather than adding the `_load_entrypoints` line to this method, would a separate `_load_entrypoints` method make sense?  This is the only method that returns a tuple

Reply:
Long term taxonomyPackage.xml should drive the entire loading process (there are links to the concept detail files in taxonomyPackage.xml) which is why I structured the code this way.  Unfortunately taxonomyPackage.xml is currently out of date and is pointing to ../documents/solar-MasterLease_2019-02-27_r01.xsd instead of correct file which is solar-MasterLease_2019-09-20.xsd so I had to scrap this part of the implementation until taxonomyPackage.xml is fixed.  Thus for the time being I load the entrypoint descriptions and determine if they fall into Data, Documents, or Process but I still rely on the directory structure for the rest of the load.